From fb85b65beb18c1d87317f18cbc78811956080549 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Mon, 8 Sep 2014 13:18:22 -0400
Subject: [ethernet] r8169: fix the incorrect tx descriptor version

Message-id: <1410182312-19560-4-git-send-email-ivecera@redhat.com>
Patchwork-id: 91996
O-Subject: [RHEL7.1 PATCH 03/13] r8169: fix the incorrect tx descriptor version
Bugzilla: 1139247
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: John Linville <linville@redhat.com>

BZ: #1139247

Upstream commit(s):
commit f75761b6b5bf6277296505941d2dd8e11f9b5c35
Author: hayeswang <hayeswang@realtek.com>
Date:   Tue Mar 11 15:11:59 2014 +0800

    r8169: fix the incorrect tx descriptor version

    The tx descriptor version of RTL8111B belong to RTL_TD_0.

    Signed-off-by: Hayes Wang <hayeswang@realtek.com>
    Acked-by: Francois Romieu <romieu@fr.zoreil.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/realtek/r8169.c b/drivers/net/ethernet/realtek/r8169.c
index e977965..3ff7bc3 100644
--- a/drivers/net/ethernet/realtek/r8169.c
+++ b/drivers/net/ethernet/realtek/r8169.c
@@ -209,7 +209,7 @@ static const struct {
  [RTL_GIGA_MAC_VER_16] =
   _R("RTL8101e",  RTL_TD_0, NULL, JUMBO_1K, true),
  [RTL_GIGA_MAC_VER_17] =
-  _R("RTL8168b/8111b", RTL_TD_1, NULL, JUMBO_4K, false),
+  _R("RTL8168b/8111b", RTL_TD_0, NULL, JUMBO_4K, false),
  [RTL_GIGA_MAC_VER_18] =
   _R("RTL8168cp/8111cp", RTL_TD_1, NULL, JUMBO_6K, false),
  [RTL_GIGA_MAC_VER_19] =
-- 
1.7.1